packages/wallets/src/components/Base/ProgressBar/__tests__/ProgressBar.spec.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import ProgressBar from '../ProgressBar';
+
+describe('ProgressBar', () => {
+    const mockProps: React.ComponentProps<typeof ProgressBar> = {
+        activeIndex: 1,
+        indexes: ['101', '102', '103'],
+        setActiveIndex: jest.fn(),
+    };
+
+    it('should render properly with list of indexes', () => {
+        render(<ProgressBar {...mockProps} />);
+        const items = screen.getAllByTestId('dt_progress_bar_item');
+        expect(items).toHaveLength(3);
+    });
+
+    it('should render the active item correctly', () => {
+        render(<ProgressBar {...mockProps} />);
+        const items = screen.getAllByTestId('dt_progress_bar_item');
+        expect(items[0]).toHaveClass('wallets-progress-bar-active');
+        items.forEach((element, index) => {
+            if (index !== 0) {
+                expect(element).toHaveClass('wallets-progress-bar-inactive');
+            }
+        });
+    });
+
+    it('should have proper class in Transition', () => {
```

Comment:
hmm, what does "proper class" mean? 
lets say that I'm running tests, and I see this test failed - how do I know whats the actually broken feature?
descriptions should act as a business documentaion, so ideally, they should be derived from features / UI. 
for example, "supports animated version of progress bar properly"